include/udx.h

```diff
@@ -362,11 +382,14 @@ udx_check_timeouts (udx_t *handle);
 int
 udx_stream_init (udx_t *udx, udx_stream_t *handle, uint32_t local_id, udx_stream_close_cb close_cb);
 
+// todo: remove these?
```

Comment:
Could we leave the getter and remove the setter?

Reply:
done